### DISCUSSION TITLE
delete invalidated consent

### DISCUSSION THREAD

**Discussion by ghost** [joomla/joomla-cms#47166] [2. Cold Features]
### Steps to reproduce the issue

try to delete invalidated consent in joomla

### Expected result

option to delete invalidated consent

### Actual result

no option to delete invalidated consents

### System information (as much as possible)

j 3.9.18

**Comment by zero-24** [comment]
Hi, what would be an invalid consent?

**Comment by ghost** [comment]
Hi, look at 

Privacy: Consents

Invalidate Consent

**Comment by zero-24** [comment]
Didn't you have to store that information for that data export thing?

**Comment by zero-24** [comment]
Why did you closed this issue? I have just raised the question whether that is a requirement. I don't know for sure. :-)

**Comment by ghost** [comment]
I don't know whether that is a requirement :-)
I can reopen this issue.

**Comment by zero-24** [comment]
Done. Lets wait whether someone can help us here maybe @jeckodevelopment can give us a hint on that question?
